scripts/update_source_data_schema_changelog.py

```diff
@@ -0,0 +1,198 @@
+                "type_changed": type_changed,
+            }
+
+    return schema_changes
+
+
+def get_print_label_string(label: str) -> str:
+    return f"\n## {label}:\n" if label else label
+
+
+def get_print_schemas_string(label="", schemas=[]) -> str:
+    print_string = ""
+    if not len(schemas):
+        return print_string
+
+    print_string += get_print_label_string(label)
+    for schema in schemas:
+        print_string += f"- {schema.replace('_schema.json', '')}"
+    return print_string
+
+
+def get_print_schema_changes_string(label="", schema_changes={}) -> str:
+    print_string = ""
+    if not schema_changes:
+        return print_string
+
+    print_string += get_print_label_string(label)
+
+    markdown_table = "|       Table Name                | Operation     | Columns                  |\n"
+    markdown_table += "|---------------------------------|---------------|--------------------------|\n"
+
+    for schema_name, operations in schema_changes.items():
+        for operation, columns in operations.items():
+            if len(columns):
+                if operation in ["column_added", "column_removed"]:
+                    columns_str = ", ".join(columns)
+                if operation in ["type_changed"]:
+                    columns_str = ", ".join(
+                        [
+                            f"{column[0]} ({column[1]} -> {column[2]})"
+                            for column in columns
+                        ]
+                    )
+                markdown_table += f"| {get_print_schemas_string(schemas=[schema_name]):<33} | {operation:<15} | {columns_str:<50} |\n"
+    print_string += markdown_table
+    return print_string
+
+
+def generate_changelog(schemas_added=[], schemas_deleted=[], schemas_changes={}) -> str:
+    new_changelog = ""
+    if schemas_added or schemas_deleted or schemas_changes:
+        current_date = datetime.now().strftime("%Y-%m-%d")
+        new_changelog += get_print_label_string(current_date)
+
+    new_changelog += get_print_schemas_string(
+        label="Tables Added", schemas=schemas_added
+    )
+    new_changelog += get_print_schemas_string(
+        label="Tables Deleted", schemas=schemas_deleted
+    )
+
+    sorted_schema_changes = sort_schema_changes(schemas_changes)
+    new_changelog += get_print_schema_changes_string(
+        label="Schema Changes", schema_changes=sorted_schema_changes
+    )
+    return new_changelog
+
+
+def main():
+    existing_changelog = read_file(filepath=CHANGELOG_FILEPATH)
+    old_schema_filepaths = list_files_in_dir(directory=OLD_SCHEMAS_DIR)
+    new_schema_filepaths = list_files_in_dir(directory=NEW_SCHEMAS_DIR)
+
+    common, added, deleted = compare_lists(
+        old_list=old_schema_filepaths, new_list=new_schema_filepaths
+    )
+    schema_changes = compare_schemas(common)
+    new_changelog = generate_changelog(
+        schemas_changes=schema_changes, schemas_added=added, schemas_deleted=deleted
```

Comment:
```suggestion
        schemas_added=added, schemas_deleted=deleted, schemas_changes=schema_changes
```

nit: make order consistent with function params